woocommerce/woocommerce-android
# Save store phone number for shipping labels

Fixes woocommerce/woomobile-private#280.

This PR saves the store phone number after it is entered in the origin address form during shipping label creation. This way, the user won't have to enter it again for a particular site because it'll be prefilled automatically.

**To test:**
1. Log in to a site that has the Shipping plugin
2. Go to Orders -> Select an eligible order (US origin & destination)
3. Tap on Create shipping label button
4. Tap on Continue
5. Notice you that a validation error is show for the origin address (missing phone number)
6. Enter a 10-digit phone number and tap done
7. You can finish the shipping label creation or you can just exit
8. Tap on the shipping label creation button again
9. Tap on continue
10. Notice the origin address is automatically validated
11. Tap on the Edit button of the origin address step
12. Notice the phone number is pre-filled
13. If you want, you can log out and log in to a different site and notice that the phone number is not pre-filled until the steps above are repeated

cc: @anitaa1990 @pmusolino
